versioned_docs/version-8.0/guides/update-guide/connectors/010-to-020.md

```diff
@@ -0,0 +1,173 @@
+---
+id: 010-to-020
+title: Update 0.1 to 0.2
+description: "Review which adjustments must be made to migrate from Connector SDK 0.1.x to 0.2.0."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from
+[Connector SDK](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md)
+0.1.x to 0.2.0.
+
+:::caution
+
+Be aware that the update from 0.1 to 0.2 requires manual migration steps as described below.
+
+:::
```

Comment:
Is it something custom? Never saw this before.

Reply:
Are we talking about the syntax or the content here? 😄 
The syntax is [Docusaurus](https://docusaurus.io/docs/markdown-features/admonitions) in the docs here 🙂

Reply:
Sorry, I meant syntax :)
Thanks for confirming!